src/common/generator.js

```diff
@@ -0,0 +1,62 @@
+const faker = require('@faker-js/faker');
+const CATEGORY = ["expenses", "income"];
+
+const random = (array) => {
```

Comment:
this should be a method inside an array.utils.js file. export it from there and import it here. the utils files should sit in a utils folder alongside common, components etc.